src/utils/randomColor.ts

```diff
@@ -1,3 +1,3 @@
 export default function () {
-  return `#${((Math.random() * 0xffffff) << 0).toString(16)}`;
+  return `#${(Math.floor(Math.random() * (0xFFFFFF - 0x100000 + 1)) + 0x100000).toString(16)}`;
```

Comment:
What's the exact bug you're trying to fix? 
are you sure this still allows for the full range of colors?

unless i'm mistaking (and based on my testing) - this won't allow for any 0x0..... codes anymore.

if it's for "too short" values (which seem to be possible with very low values) - maybe 
`Math.floor(Math.random() * 0xffffff).toString(16).padStart(6, '0')` is a better solution?